Include Partition in Default Scope - release notes (#4125)
SumoLogic/sumologic-documentation: docs/manage/partitions/flex/flex-pricing-faq.md

We recommend you configure partitions to have less than 5 TB per day flowing into them. This practice:

* Reduces the cost of each query.
* Results in faster queries, since less data has to be scanned.  

## How to optimize the query using default scope?

> Default scope allows you to include or exclude the partitions in the search query, helping you to optimize the search cost. For example, consider if a query needs to run through 10 partitions which consumes about 10 GB of search data, narrowing the search query using the default scope can reduce the time and cost. You can achieve this by selecting the **Include this partition in default scope** checkbox when creating the partition. Let's say, out of 10 partitions you excluded two partitions, now, when you run the query without any `_index`, the search will only consider the included partitions, reducing the amount of data scanned and lowering the cost.

Reviewer: selecting or deselecting

Author: Thanks for your feedback Krati, I have fixed your comments.

> Default scope allows you to include or exclude the partitions in the search query, helping you to optimize the search cost. For example, consider if a query needs to run through 10 partitions which consumes about 10 GB of search data, narrowing the search query using the default scope can reduce the time and cost. You can achieve this by selecting or deselecting the **Include this partition in default scope** checkbox when creating the partition. Let's say, out of 10 partitions you excluded two partitions, now, when you run a query which does not have `_index`, the search will only consider the included partitions, reducing the amount of data scanned and lowering the cost.

Reviewer 2: ```suggestion
Default scope allows you to include or exclude the partitions in the search query, helping you to optimize the search cost. For example, consider if a query needs to run through 10 partitions which consumes about 10 GB of search data, narrowing the search query using the default scope can improve the query performance and scan cost. You can modify the default scope by selecting or deselecting the **Include this partition in default scope** checkbox when creating/updating the partition. Let's say, out of 10 partitions you excluded two partitions, now, when you run a query which does not have `_index` / `_view` term referenced in the query, the search will only consider the included partitions, reducing the amount of data scanned and lowering the cost.
```

> Default scope allows you to include or exclude the partitions in the search query, helping you to optimize the search cost. For example, consider if a query needs to run through 10 partitions which consumes about 10 GB of search data, narrowing the search query using the default scope can improve the query performance and scan cost. You can modify the default scope by selecting or deselecting the **Include this partition in default scope** checkbox when creating/updating the partition. Let's say, out of 10 partitions you excluded two partitions, now, when you run a query which does not have `_index` / `_view` term referenced in the query, the search will only consider the included partitions, reducing the amount of data scanned and lowering the cost.

Reviewer 3: ```suggestion
Default scope allows you to include or exclude the partitions in the search query, helping you to optimize the search cost. For example, consider if a query needs to run through 10 partitions which consumes about 10 GB of search data, narrowing the search query using the default scope can improve the query performance and scan cost. You can modify the default scope by selecting or deselecting the **Include this partition in default scope** checkbox when creating/updating the partition. Let's say, out of 10 partitions you excluded two partitions. Now, when you run a query which does not have `_index` / `_view` term referenced in the query, the search will only consider the included partitions, reducing the amount of data scanned and lowering the cost.
```

> When partitions are marked as included and `_index` is not referred to in the query, all the included partitions will be considered for the search query by default. Default scope can also come to use when `AND` or `OR` conditions are referred to in the query with `_index`. For example, consider you have three partitions namely, Partition A (Excluded), Partition B (Included), and Partition C (Included). Below are some scenarios:

Reviewer 2: ```suggestion
When partitions are marked as included and `_index`/`_view` is not referenced in the query, all the included partitions will be considered for the search query by default. Default scope can also come to use when `AND` or `OR` conditions are referred to in the query with `_index`. For example, consider you have three partitions namely, Partition A (Excluded), Partition B (Included), and Partition C (Included). Below are some scenarios:
```

> When partitions are marked as included and `_index`/`_view` is not referenced in the query, all the included partitions will be considered for the search query by default. Default scope can also come to use when `AND` or `OR` conditions are referred to in the query with `_index`. For example, consider you have three partitions namely, Partition A (Excluded), Partition B (Included), and Partition C (Included). Below are some scenarios:

Reviewer 3: ```suggestion
When partitions are marked as included and `_index`/`_view` is not referenced in the query, all the included partitions will be considered for the search query by default. Default scope can also come into use when `AND` or `OR` conditions are referred to in the query with `_index`. For example, consider you have three partitions namely, Partition A (Excluded), Partition B (Included), and Partition C (Included). Below are some scenarios:
```

> - When you run the query without referring to `_index`, only Partition B and Partition C will be considered for the query. 

Reviewer 2: ```suggestion
- When you run the query without referring to `_index` eg. `error | count`, only Partition B and Partition C will be considered for the query. 
```

> - When you run a query referring to `_index=Partition A`, only Partition A will be considered for the query. 

Reviewer 2: ```suggestion
- When you run a query referring to an index term e.g. `_index="Partition A"`, only Partition A will be considered for the query. 
```

Space is not allowed in the partition name.

> - When you run the query without referring to `_index`, example: `error | count`, only Partition B and Partition C will be considered for the query, as Partition A is excluded from the default scope. 

Reviewer 3: ```suggestion
- When you run the query without referring to `_index`, for example `error | count`, only Partition B and Partition C will be considered for the query, as Partition A is excluded from the default scope. 
```

- When you run the query, `_index=Partition A OR _index=Partition B`, only Partition A and Partition B will be considered for the query. 

> - When you run the query, `_index=Partition A OR error`, Partition A, Partition B, and Partition C will be considered for the query. 

Reviewer 2: ```suggestion
- However, When you run the query, `_index="Partition A" OR error`, Partition A, Partition B, and Partition C will be considered for the query, because the error keyword might be present in Partition B or C as well. So in cases, like this, we need to scan all three of them. 
```

> - When you run the query without referring to `_index`, example: `error | count`, only Partition B and Partition C will be considered for the query, as Partition A is excluded from the default scope. 
> - When you run a query referring to an index term, example: `_index="Partition A"`, only Partition A will be considered for the query. 
> - Similarly, When you refer to multiple index terms, example: `_index="Partition A" OR _index="Partition B"`, only Partition A and Partition B will be considered for the query. 
> - However, When you run the query, example: `_index="Partition A" OR error`, all of the partitions - Partition A, Partition B, and Partition C will be considered for the query, because the error keyword might be present in Partition B or C as well. So in cases like this, we need to scan all three of them. 

Reviewer 3: ```suggestion
- When you run the query without referring to `_index`, for example `error | count`, only Partition B and Partition C will be considered for the query, as Partition A is excluded from the default scope. 
- When you run a query referring to an index term, for example `_index="Partition A"`, only Partition A will be considered for the query. 
- Similarly, when you refer to multiple index terms, for example `_index="Partition A" OR _index="Partition B"`, only Partition A and Partition B will be considered for the query. 
- However, when you run the query, for example `_index="Partition A" OR error`, all of the partitions (Partition A, Partition B, and Partition C) will be considered for the query, because the error keyword might be present in Partition B or C as well. So in cases like this, we need to scan all three of them. 
```
